### PR TITLE
Make workflows selectable

### PR DESCRIPTION
**Pull request by mweidling**
This PR (re-)adds a CLI to the project for easier usage.

Furthermore, it allows users to specify which workflows are run on the data they provide in `./gt`.

### CODE REVIEW

File: quiver [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        ;;
+    prepare-default-gt)
+        docker compose exec app bash scripts/prepare.sh
+        ;;
+    run)
+        echo "Starting to process and collect benchmarks."
+        DATE=$(date +"%F-%H:%M:%S")
+        echo "See logs/run_$DATE.log for more specific info."
+        mkdir -p logs
+	    docker compose exec app quiver run-ocr > logs/run_"$DATE".log
```
Comment (stweil) [inline quiver:18]:
```suggestion
        mkdir -p logs
        docker compose exec app quiver run-ocr > logs/run_"$DATE".log
```

File: quiver [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        exit 0
+        ;;
+    start)
+        echo "Starting up a container for QuiVer. ID:"
+        docker compose run -d app
+        ;;
+    stop)
+        echo "Stopping all running QuiVer containers."
+        CONTAINER_ID=$(docker ps | grep quiver | cut -d' ' -f1)
+        docker container stop "$CONTAINER_ID" && docker container remove "$CONTAINER_ID"
```
Comment (stweil) [inline quiver:29]:
```suggestion
        docker container stop "$CONTAINER_ID" && docker container rm "$CONTAINER_ID"
```

File: quiver [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        ;;
+    *)
+        echo "Error: Unrecognized command. Please refer to the documentation for all functionality."
+        exit 1
+        ;;
+esac
```
Comment (stweil) [inline quiver:35]:
```suggestion
esac
```

**Comment by stweil** [comment]
Are there plans to merge this pull request? I have local changes (support for podman) pending which would cause merge conflicts with the changes here.

**Comment by mweidling** [comment]
> Are there plans to merge this pull request? I have local changes (support for podman) pending which would cause merge conflicts with the changes here.

No, I don't think so. This branch has been stale for quite some time.

**Pull request closed without merging**